workspaces/keycloak/plugins/catalog-backend-module-keycloak/src/lib/read.ts

```diff
@@ -257,97 +283,142 @@ export const readKeycloakRealm = async (
 
   let rawKGroups: GroupRepresentationWithParent[] = [];
 
+  logger.info(`Processing groups recursively`);
```

Comment:
Do we still need all of these loggers?

Reply:
No, but for me this log was good indicator. Maybe users might want to see what happening during log operations... We can change log level from info to debug. What do you think?

Reply:
Debug sounds good. Just worried that this would lead to a lot of unnecessary logs when everything is working as intended.

Reply:
Done.